An error occurred while executing the following cell:
------------------
# Install Augmax for Image Augmentation
try:
    import augmax
except ModuleNotFoundError:
    %pip install -qq git+https://github.com/khdlr/augmax.git -q
    import augmax

# Install the  jax-resnet
try:
    import jax_resnet
except ModuleNotFoundError:
    %pip install -qq git+https://github.com/n2cholas/jax-resnet.git -q
    import jax_resnet
------------------

---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
/tmp/ipykernel_4857/3209891542.py in <module>
      9 try:
---> 10     import jax_resnet
     11 except ModuleNotFoundError:

ModuleNotFoundError: No module named 'jax_resnet'

During handling of the above exception, another exception occurred:

AttributeError                            Traceback (most recent call last)
/tmp/ipykernel_4857/3209891542.py in <module>
     11 except ModuleNotFoundError:
     12     get_ipython().run_line_magic('pip', 'install -qq git+https://github.com/n2cholas/jax-resnet.git -q')
---> 13     import jax_resnet

~/miniconda3/envs/py37/lib/python3.7/site-packages/jax_resnet/__init__.py in <module>
      1 from .common import *  # noqa
----> 2 from .pretrained import *  # noqa
      3 from .resnet import *  # noqa
      4 from .splat import *  # noqa
      5 

~/miniconda3/envs/py37/lib/python3.7/site-packages/jax_resnet/pretrained.py in <module>
     13 
     14     # https://github.com/pytorch/vision/issues/4156#issuecomment-894768539
---> 15     torch.hub._validate_not_a_forked_repo = lambda *_: True  # type: ignore
     16     torch_exists = True
     17 except ImportError:

AttributeError: module 'torch' has no attribute 'hub'
AttributeError: module 'torch' has no attribute 'hub'
